src/models/team.rs

```diff
@@ -116,6 +116,7 @@ impl Team {
         // check that `team` is the `slug` in results, and grab its data
 
         // "sanitization"
+        #[cfg_attr(feature = "cargo-clippy", allow(trivially_copy_pass_by_ref))]
```

Comment:
wdyt about, instead, changing this to `fn whitelist(c: char) -> bool` and line 126 to `if let Some(c) = org_name.chars().find(|c| whitelist(*c)) {`?